design-proposals/network-binding-plugin/network-binding-plugin.md

```diff
@@ -1162,4 +1208,53 @@ Libvirt configuration snippet:
 </interface>
 ```
 
-- The domain itself is created by the `virt-launcher`.
\ No newline at end of file
+- The domain itself is created by the `virt-launcher`.
+
+# Appendix G: Alternatives to plugin sidecar container resource specification
+
+1. The cluster admin could configure worst case CPU/Memory requests and/or limits on the KubeVirt CR:
+
+Pros:
+- Already implemented.
+
+Cons:
+- Applies uniformly on all hook sidecars and network binding plugins
+    - Worst case CPU/Memory requests/limits should be defined on the KubeVirt CR which maybe wasteful for a combination of hook sidecars and network binding plugins.
+    - Coarse level of control.
+    - Only supports CPU and memory requests and limits.
+
+2. Additional API for network binding plugin sidecar resource specification:
+
+The network binding plugin API in the KubeVirt CR could receive an additional input field to specify the sidecar resource requirements:
+
+```yaml
+apiVersion: kubevirt.io/v1
+kind: KubeVirt
+metadata:
+  name: kubevirt
+  namespace: kubevirt
+spec:
+  configuration:
+    network:
+      binding:
+        mynetbindingplugin:
+          sidecarImage: quay.io/kubevirt/mynetbindingplugin
+          sidecarResources:
+            requests:
```

Comment:
Maybe it better to use the native resources API as in Pod [1] 

[1] https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#example-1

Reply:
IIUC this is exactly what was suggested here.

Reply:
This is indeed what is proposed here.

Reply:
This is a mere yaml example it can be anything :slightly_smiling_face: